CMakeLists.txt

```diff
@@ -691,8 +732,12 @@ if(APPLE)
     " BUNDLE DESTINATION ${CMAKE_BINARY_DIR})
 endif()
 
-include_directories(${SDL2_INCLUDE_DIR})
-include_directories(${SDL2_MIXER_INCLUDE_DIR})
+if(SDL2_INCLUDE_DIR)
+    include_directories(${SDL2_INCLUDE_DIR})
```

Comment:
In which case would the include dir be empty? Where do the SDL includes come from in that case?

Reply:
When the new variable `EMSCRIPTEN_LOAD_SDL_PORTS` (which I added to `CMakeLists.txt`) is set, emscripten will be run with the options `-s  USE_SDL=2 -s USE_SDL_MIXER=2`, which will automatically download and include them without needing to configure their inclusion on cmake.

This is useful if the user wants to compile Julius for emscripten and not have the hassle of compiling libmpg123, SDL and SDL_mixer, which are kinda hard to configure for emscripten. However,  the default ports don't have mp3 support, which our manual compilation does.